src/cljam/io/sam/util.clj

```diff
@@ -77,6 +77,19 @@
         (map parse-header-line)
         into-header)))
 
+(defn- ^"[B" as-byte-array [^String s]
```

Comment:
proton has `hex->bytes` function, which seems to be equivalent of this function. Is any reason to reinvent it?